server/chat/backend/agent/workflow.py

```
@@ -1743,6 +1747,47 @@ def _append_new_turn_ui_messages(
+        if not text or not _guardrails_config.output_redaction:
+            return text
+        try:
+            t0 = time.perf_counter()
+            redacted, findings = _l5_redact(text)
+            if not findings:
+                return redacted
+            latency_ms = (time.perf_counter() - t0) * 1000.0
+            session_id = ""
+            try:
+                session_id = self.config["configurable"]["thread_id"]
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)